.ruby-version

```diff
@@ -1 +1 @@
-2.4.1
+2.4.0
```

Comment:
Let's remove this change. I'm actually going to gitignore the .ruby-version file and remove it in another commit